OpenRA.Game/Server/Server.cs

```diff
@@ -220,8 +220,8 @@ public Server(List<IPEndPoint> endpoints, ServerSettings settings, ModData modDa
 			if (type != ServerType.Local && settings.EnableGeoIP)
 				GeoIP.Initialize();
 
-			if (type != ServerType.Local && UPnP.Status == UPnPStatus.Enabled)
-				UPnP.ForwardPort(Settings.ListenPort, Settings.ListenPort).Wait();
+			if (type != ServerType.Local && Game.Nat.Status == NatStatus.Enabled)
```

Comment:
```
Unhandled exception. System.NullReferenceException: Object reference not set to an instance of an object.
   at OpenRA.Server.Server..ctor(List`1 endpoints, ServerSettings settings, ModData modData, ServerType type) in /Users/paul/src/OpenRA/OpenRA.Game/Server/Server.cs:line 223
   at OpenRA.Server.Program.Main(String[] args) in /Users/paul/src/OpenRA/OpenRA.Server/Program.cs:line 76
```

I was politely asking about the dedicated server and suggesting the `Game.UPnPStatus` wrapper earlier because I was sure that this was going to crash. Testing indeed confirmed this.

Reply:
I am politely saying that I tested it, but not after applying other changes you requested.

Reply:
The good thing about my sloppiness is that it showed that dedicated NAT discovery was apparently broken. There is a log file for it, so I guess this was not purposefully disabled.

Reply:
> I guess this was not purposefully disabled

More so known to be broken and purposely never fixed. UPnP doesn't really work for dedicated network services, so trying to support dedicated servers opens up new headaches (such as working around the 10 hour timeout).

IMO it really isn't worth your time to try and fix the extra issues the dedicated server brings, or our time to try and test them, so I suggest we instead change this to `type == ServerType.Multiplayer` and add a comment documenting that this is deliberately disabled.